### PR TITLE
Force prettier rules

### PR DESCRIPTION
**Pull request by Pomax**
Closes https://github.com/mozilla/foundation.mozilla.org/issues/5289 by adding `prettier ... --write` for all js/jsx/scss tasks, so that by the time esline/stylelint run, it is impossible for them to find _style_ errors, they'll only find _actual_ errors (if there are any), because prettier can't spot those: it simply doesn't run on files with parsing errors.

I've added these tasks as part of `npm test`, so that they also run as part of `inv test` because of `inv test-node`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,8 +35,11 @@
     "server": "python network-api/manage.py runserver",
     "start": "docker-compose up",
     "test:procfile": "node test/test-procfile.js",
+    "test:eslint:js:pre": "prettier \"cypress/integration/*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js --write",
     "test:eslint:js": "eslint --config ./.eslintrc.json \"cypress/integration/*.js\" \"source/js/**/*.js\" \"source/js/**/*.jsx\" webpack.config.js",
+    "test:eslint:a11y:pre": "prettier \"source/js/**/*.jsx\" --write",
     "test:eslint:a11y": "eslint --config ./.eslintrc.a11y.json \"source/js/**/*.jsx\"",
+    "test:scss:pre": "prettier \"source/sass/**/*.scss\" \"source/js/**/*.scss\" \"network-api/networkapi/{,!(frontend)/**/}*.css\" --write",
```
Comment (mmmavis) [inline package.json:42]: do we need to care about `.css` files?

Comment (KalobTaulien) [reply package.json:42]: I thought it was `pre` as in "run before something else"` but then I saw the prettier command and dont know which is which now. Could we make `:pre` something more explicit, possibly?

Comment (Pomax) [reply package.json:42]: @mmmavis we have a few bare .css files in overrides, we might as well format them

@KalobTaulien no, you got that right: this runs "pre" the actual `test:scss` running, blindingly applying code formatting before running code _linting_

Comment (KalobTaulien) [reply package.json:42]: Perfect. Thanks for adding that clarification Pomax!

**Review by mmmavis — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mmmavis) [reply package.json:42]: Ah I forgot we have a few `.css`. Good call including them.

**Pull request merged**